src/Microsoft.Framework.Caching.SqlServer/project.json

```diff
@@ -0,0 +1,29 @@
+{
+    "commands": {
+        "sqlservercache": "Microsoft.Framework.Caching.SqlServer"
+    },
+    "dependencies": {
+        "Microsoft.Framework.Caching.Abstractions": "1.0.0-*",
+        "Microsoft.Framework.Logging": "1.0.0-*",
+        "Microsoft.Framework.Logging.Abstractions": "1.0.0-*",
```

Comment:
Hmm I just noticed that both the command line util and the provider itself seem to be in the same package. Should those not be split out into separate packages? Their functionality probably doesn't overlap a whole lot. And their usage is certainly orthogonal. One is a utility used during deployment, the other is a runtime feature. They're probably never even used on the same machine, except during development.